### PR TITLE
rtfctl examples

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -29,6 +29,25 @@ To view a list of all supported commands, run `rtfctl -h`.
...
+
+.. Make sure to base64 encode the license following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
+.. Run the following command:
++
+----
+$ sudo ./rtfctl apply mule-license '<license-key-information>'
+Updating rtf namespace... OK
+----
+
+* Collect debug package information for an application
```
Comment (sundippatel) [inline modules/ROOT/pages/install-rtfctl.adoc]: I think this section belongs in another topic -- possibly troubleshooting.

**Review by sundippatel — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-rtfctl.adoc]: Good point - I've moved this. Please check to make sure my intro sentence is correct. Thank you! My big plan is to go through all the content and reorganize it. There are other cases of information needing to be moved IMHO.

**Review by sundippatel — APPROVED** [review]

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -29,6 +29,18 @@ To view a list of all supported commands, run `rtfctl -h`.
 
 To get more information on a specific command, run `rtfctl <command> -h`.
 
+== Usage
+
+* Install a Mule license
+
+.. Make sure to base64 encode the license following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
```
Comment (dhirsch1) [inline modules/ROOT/pages/install-rtfctl.adoc:36]: I would add a hyphen between bse64 and encode.

**Review by dhirsch1 — APPROVED** [review]
Approved with one comment.

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-rtfctl.adoc:36]: Thank you! I know it's weird - it's actually called base64 encode.

**Pull request merged**